docs-content/general/7_integrations/wordpress-integration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: WordPress Plugin
+slug: wordpress-plugin
+createdAt: 2024-10-01T12:00:00.000Z
+updatedAt: 2024-10-01T12:00:00.000Z
+---
+
+# Highlight.io WordPress Plugin - Session Replay, Logging, and Error Monitoring for WordPress
+
+The Highlight WordPress Plugin allows you to easily integrate Highlight's powerful session recording and monitoring capabilities into your WordPress site. With this plugin, you can gain valuable insights into user behavior, troubleshoot issues, and improve your website's performance.
+
+## Features
+
+- **Easy Setup**: Install and configure the plugin in just a few clicks.
+- **Session Recording**: Capture user interactions on your WordPress site.
+- **Error Monitoring**: Automatically detect and log errors with all the information you need to troubleshoot.
+- **Network Recording**: Record network requests for deeper insights.
+- **Console Log Recording**: Capture and review console logs to help identify and fix issues quickly.
+
+## Installation
+
+1. Log in to your WordPress admin panel.
+2. Go to "Plugins" > "Add New".
```

Comment:
maybe here we could link to the wordpress plugin list?

Reply:
I think clicking "Add New" will give them some UI to explore in the directory.